pkg/kubeapiserver/options/authentication.go

```diff
@@ -132,6 +132,8 @@ type ServiceAccountAuthenticationOptions struct {
 	JWKSURI          string
 	MaxExpiration    time.Duration
 	ExtendExpiration bool
+	// WithoutNodes is a flag to indicate that the service account token getter should be created without nodes support.
+	WithoutNodes bool
```

Comment:
this boolean doesn't make sense in kubeapiserver, where nodes and pods are always available